docs/content/StyledOcticon.md

```diff
@@ -20,9 +20,9 @@ StyledOcticon components get `COMMON` system props. Read our [System Props](/sys
 
 StyledOcticon passes all of its props except the common system props down to the [Octicon component](https://github.com/primer/octicons/tree/master/lib/octicons_react#usage), including:
 
-| Name | Type | Default | Description |
-| :- | :- | :-: | :- |
-| ariaLabel | String | | Specifies the `aria-label` attribute, which is read verbatim by screen readers |
-| icon | Component | | [Octicon component](https://github.com/primer/octicons/tree/master/lib/octicons_react) used in the component |
-| size | Number | 16 | Sets the uniform `width` and `height` of the SVG element |
-| verticalAlign | String | `text-bottom` | Sets the `vertical-align` CSS property |
+| Name          | Type      |    Default    | Description                                                                                                  |
+| :------------ | :-------- | :-----------: | :----------------------------------------------------------------------------------------------------------- |
+| ariaLabel     | String    |               | Specifies the `aria-label` attribute, which is read verbatim by screen readers                               |
+| icon          | Component |               | [Octicon component](https://github.com/primer/octicons/tree/master/lib/octicons_react) used in the component |
+| size          | Number    |      16       | Sets the uniform `width` and `height` of the SVG element                                                     |
+| verticalAlign | String    | `text-bottom` | Sets the `vertical-align` CSS property                                                                       |
```

Comment:
linter is really working overtime, but I think the extra spacing here makes it much easier to read in an IDE.